ui/pages/asset/components/asset-page.tsx

```diff
@@ -115,6 +116,10 @@ const AssetPage = ({
   const isBuyableChain = useSelector(getIsNativeTokenBuyable);
   const isEvm = useMultichainSelector(getMultichainIsEvm);
 
+  useEffect(() => {
+    endTrace({ name: TraceName.AssetDetails });
```

Comment:
Wow this is cool, trace starts in 1 file but ends here. So we are testing page loading? I like it!

Reply:
Yep!